db/migrate/20151008114043_create_chargeback_rate_detail_measure.rb

```diff
@@ -0,0 +1,12 @@
+class CreateChargebackRateDetailMeasure < ActiveRecord::Migration
+  def change
+    create_table :chargeback_rate_detail_measures do |t|
+      t.string :name
+      t.string :units, :array => true
```

Comment:
Give that this is the last week of the last sprint I would be more comfortable with avoiding using the array type. You can instead serialize it as an Array in the model:
```
serialize :units, Array
```